creator/Documents/DifferencesBetweenBedrockAndJava.md

```diff
@@ -8,95 +8,133 @@ description: "An article covering the major differences between Minecraft: Bedro
+
+For example, compare the syntax of `/summon`:
 
 ```
-summon <entityType: EntityType> [spawnPos: x y z] [spawnEvent: String] [named: String]
+Java:
+/summon <entity> [<position>] [<data>]
+
+Bedrock:
+/summon <entity> [<position>] [<event>] [<name>]
 ```
 
-Currently, there’s no way to `/give` players custom items in Bedrock Edition as you can in Java Edition. The item will need to be created beforehand and teleported to the player. The most common ways of doing this is either by placing the item in a chest and breaking the chest, or making an entity drop it on death via loot table.
+Bedrock can provide an [event to run](../Reference/Content/EntityReference/Examples/EventList.md) on the spawned mob, and a name to be applied.
+
+For giving special items, Bedrock can specify exactly four properties:
+* `can_place_on`
+* `can_destroy`
+* `item_lock`
+* `keep_on_death`
+
+Other properties such as durability, enchantments, custom names, book text, color, and more cannot be specified. As a workaround, [loot tables](../Documents/IntroductionToLootTables.md) can be used in tandem with the `/loot` command in some cases.
+
+### `/execute`
+Bedrock Edition is still using the pre-1.13 `/execute` command, with the following overloads:
+```
+/execute <entity> <position> <command>
+/execute <entity> <position> detect <position> <block> <damage value> <command>
+```
+Java's new `/execute` command, with individual subcommands like `as`, `at`, `if`, `unless`, `positioned`, `store`, `facing`, `align`, and so on is not supported.
 
-Aside from that, commands should feel very familiar between Bedrock Edition and Java Edition versions prior to 1.13. The execute format introduced in Java Edition 1.13 is not supported in Bedrock Edition.
+### Scoreboards
 
-**Scoreboards** function the same way between the two versions, but Bedrock Edition currently doesn't have support for the wide range of criteria that Java Edition does. Currently, the only criteria supported by Bedrock Edition is the `dummy` criteria. None of the other criteria available in Java Edition have been implemented by Bedrock Edition. There’s also no support for commands such as `/stats` or `/team`.
+Bedrock has support for basic scoreboard functionality.
 
-**Schedule** commands differ between editions. In Java Edition, the `/schedule` command has the following syntax:
+Unlike Java Edition, which has criteria for mob kills, deaths, and statistics such as item usage, Bedrock only supports the `dummy` criteria, which must be incremented manually with commands.
 
+The `/team` command also does not exist in Bedrock Edition.
+
+Scoreboard syntax is otherwise identical between editions. Bedrock also has one additional `/scoreboard` subcommand: `/scoreboard players random`.
+
```

Comment:
Separate scoreboard from schedule a bit and elaborate a bit more, including bringing up one of Bedrock's scoreboard advantages